### PR TITLE
Do not include example project into projects page

### PR DESCRIPTION
**Pull request by ddemidov**
The link to the project is still present on the Share page.

This is a result of irc discussion with @cavenel.

### CODE REVIEW

File: projects/index.md [hunk 1]
```diff
@@ -12,23 +12,25 @@ Would you like your project added to the list? We would!
...
-                    {% assign num_words = 20 %}
-                    <a href="{{ site.github.url }}{{ post.url }}">
-                        <img class="project-tile-img" src="http://img.youtube.com/vi/{{post.youtube_video_id}}/mqdefault.jpg" />
-                    </a>
-                {% endif %}
-                <span class="project-tile-excerpt">
-                    {{ post.excerpt | truncatewords: num_words }}
-                </span>
-            </li>
+            {% unless post.title contains "Example" %}
```
Comment (dlech) [inline projects/index.md]: What if someone else uses the word "Example" in their title? Can we make this a more exact match?

Comment (ddemidov) [inline projects/index.md]: I am not very familiar with liquid templates syntax. I tried to use `post.title != "Example Project"`, but that did not work for some reason. Do you have any suggestions?

Comment (dlech) [inline projects/index.md]: Looking at <http://jekyllrb.com/docs/variables/#page-variables>, there is a `page.id` variable that provides a unique identifier for the page. You can add `{{ page.id }}` to the page temporarily to see what the actual id is.

I don't see any reason why `==` or `!=` should not work.

**Comment by ddemidov** [comment]
060b1f4 uses post.id to exclude the example project.

**Pull request merged**